pandas/tests/arithmetic/test_period.py

```diff
@@ -582,12 +575,14 @@ def test_pi_add_offset_n_gt1(self, box):
     def test_pi_add_offset_n_gt1_not_divisible(self, box_with_period):
         # GH#23215
         # PeriodIndex with freq.n > 1 add offset with offset.n % freq.n != 0
+        box = box_with_period
 
         pi = pd.PeriodIndex(['2016-01'], freq='2M')
-        pi = tm.box_expected(pi, box_with_period)
-
         expected = pd.PeriodIndex(['2016-04'], freq='2M')
-        expected = tm.box_expected(expected, box_with_period)
+
+        # FIXME: with transposing these tests fail
```

Comment:
shouldn't you just leave all of the tests that you set ``transpose=False`` and instead just use ``box_transpose_fail``?

Reply:
ok that's fine .then.